libevm/VM.cpp

```diff
@@ -573,6 +573,48 @@ void VM::interpretCases()
 		}
 		NEXT
 
+#ifdef EVM_USE_BITSHIFT
+		CASE(SHL)
+		{
+			ON_OP();
+			updateIOGas();
+
+			/// This workarounds a bug in Boost, ...
+			u256 mask = (u256(1) << (256 - m_SP[1])) - 1;
```

Comment:
The EIP says `If the shift amount (`arg2`) is greater or equal 256 the result is 0.` so the above code should be updated accordingly.